distributed/tests/test_client.py

```diff
@@ -301,17 +301,21 @@ def test_gather_strict(c, s, a, b):
 
 @gen_cluster(client=True, timeout=None)
 def test_get(c, s, a, b):
-    result = yield c._get({'x': (inc, 1)}, 'x')
+    result = c.get({'x': (inc, 1)}, 'x', sync=False)
+    assert isinstance(result, Future)
+    result = yield result
```

Comment:
Request using the pronoun `future` or `futures` for futures and `result` for concrete results.